### PR TITLE
fix the broken links in the documentation

### PR DESCRIPTION
**Pull request by krunalsshah**
fix the broken links in the documentation

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -50,7 +50,6 @@ Licensed under the [Apache License, Version 2.0].
 
 [gitbook]: https://www.gitbook.com/
 
-[what is electrode native]: ./overview/what-is-ern.md
+[What is electrode native]: ./overview/what-is-ern.md
```
Comment (friederbluemle) [inline docs/README.md]: Also capitalize "Electrode Native"?

File: docs/cli/bundlestore.md [hunk 2]
```diff
@@ -84,4 +84,4 @@ Disabling the bundle store will automatically restore settings that were in use
 [bundlestore use]: ./bundlestore/use.md
 [bundlestore delete]: ./bundlestore/delete.md
 [bundlestore upload]: ./bundlestore/upload.md
-[Electrode Native bundle store server]: https://github.com/electrode-io/ern-bundle-store
\ No newline at end of file
+[Electrode Native bundle store server]:https://github.com/electrode-io/ern-bundle-store
```
Comment (friederbluemle) [inline docs/cli/bundlestore.md]: No need to remove the space after `:` here. (probably unintentional?)

File: docs/README.md (hunk 1 above)
Comment (friederbluemle) [reply docs/README.md]: Oops wait, I just realized it refers to a markdown link. Ignore my comment

File: docs/overview/what-is-ern.md [hunk 3]
```diff
@@ -74,4 +74,4 @@ An Electrode Native runner application is automatically generated the first time
 
 [Learn More about the Runner](../platform-parts/runner/index.md)
 
-[Electrode Native Bridge]:https://github.com/electrode-io/react-native-electrode-bridge
\ No newline at end of file
+[Electrode Native Bridge]:https://github.com/electrode-io/react-native-electrode-bridge
```
Comment (friederbluemle) [inline docs/overview/what-is-ern.md]: What changed here❓

File: docs/README.md (hunk 1 above)
Comment (friederbluemle) [reply docs/README.md]: Actually, it still applies. The link target also has "Electrode Native" capitalized. Would be good to keep it the same.

Comment (krunalsshah) [reply docs/README.md]: 👍

File: docs/cli/bundlestore.md (hunk 2 above)
Comment (krunalsshah) [reply docs/cli/bundlestore.md]: Yes

File: docs/overview/what-is-ern.md (hunk 3 above)
Comment (krunalsshah) [reply docs/overview/what-is-ern.md]: https://www.markdownguide.org/basic-syntax/#reference-style-links

File: docs/overview/what-is-ern.md [hunk 4]
```diff
@@ -74,4 +74,4 @@ An Electrode Native runner application is automatically generated the first time
 
 [Learn More about the Runner](../platform-parts/runner/index.md)
 
-[Electrode Native Bridge]:https://github.com/electrode-io/react-native-electrode-bridge
\ No newline at end of file
+[Electrode Native Bridge]: https://github.com/electrode-io/react-native-electrode-bridge
```
Comment (krunalsshah) [inline docs/overview/what-is-ern.md]: https://www.markdownguide.org/basic-syntax/#reference-style-links
Formating

File: docs/overview/what-is-ern.md (hunk 3 above)
Comment (friederbluemle) [reply docs/overview/what-is-ern.md]: So weird.. I was pretty sure I commented on the last line in `bundlestore.md` - The removed line and the added line look exactly the same (here on GitHub)...

Comment (friederbluemle) [reply docs/overview/what-is-ern.md]: Checking the commit content on command line revealed that you added the missing newline at end of file 👍

**Pull request merged**